### ISSUE TITLE
No way to view log in Homey V2.0.0-rc1

### ISSUE THREAD

**Issue by AdyRock** [irritanterik/homey-logger.nsa.gov#6]
In Homey 2.0.0-rc1 there is no way I can find to setup or see the logged data.

**Comment by irritanterik** [comment]
App is not yet v2 compatible. Not sure if it will be possible to get compatible.

**Comment by AdyRock** [comment]
I hope it can be made to work as it has been very useful in the past.

**Comment by irritanterik** [comment]
Yes, will test it when v2 is somewhat stable. Running my house on Homey, so will not run experimental versions on it.

**Comment by AdyRock** [comment]
Fair enough. I will keep my fingers crossed and watch for updates if you can.
